release/serve_tests/workloads/multi_deployment_1k_noop_replica.py

```diff
@@ -143,13 +143,11 @@ def main(num_replicas: Optional[int], num_deployments: Optional[int],
     setup_multi_deployment_replicas(num_replicas, num_deployments)
 
     logger.info("Warming up cluster ....\n")
-    rst_ray_refs = []
     all_endpoints = list(serve.list_deployments().keys())
     for endpoint in all_endpoints:
-        rst_ray_refs.append(
+        ray.get(
```

Comment:
is this an attempt to limit concurrent cluster warm up by doing it one at a time ?

Reply:
yup